setup.py

```diff
@@ -395,7 +395,7 @@ def finalize_options(self):
         NUMPY_STR,
         'scipy >= 0.18.1',
         'six >= 1.5.0',
-        'smart_open >= 1.8.1',
+        'smart_open >= 1.8.1, < 1.11',  # smart_open >= 1.11 is py3+ only. TODO: Remove once we drop py2.7 from gensim too.
```

Comment:
We've already dropped Py2.7 support for gensim on the develop branch. So, this change is not required.